tests/neuro_san/coded_tools/music_nerd_pro/test_accountant.py

```diff
@@ -1,14 +1,17 @@
 from unittest import TestCase
 
-from neuro_san.coded_tools.music_nerd_pro.accountant import Accountant
+import pytest
+
+from neuro_san.coded_tools.accountant import Accountant
 
 
 class TestAccountant(TestCase):
     """
     Unit tests for Accountant class.
     """
 
-    def test_invoke(self):
+    @pytest.mark.asyncio
+    async def test_invoke(self):
```

Comment:
Now that the accountant is async, we need a way to test it asynchronously.
This is what the docs say to do with this pytest.mark.asyncio biz.